central/notifiers/microsoftsentinel/sentinel.go

```diff
@@ -41,6 +44,26 @@ type sentinel struct {
 	azlogsClient azureLogsClient
 }
 
+func (s sentinel) SendAuditMessage(ctx context.Context, msg *v1.Audit_Message) error {
+	if !features.MicrosoftSentinelNotifier.Enabled() {
+		return nil
+	}
+
+	if !s.notifier.GetMicrosoftSentinel().GetAuditLogDcrConfig().GetEnabled() {
+		return nil
+	}
+
+	err := s.uploadLogs(ctx, s.notifier.GetMicrosoftSentinel().GetAuditLogDcrConfig(), msg)
+	if err != nil {
+		return errors.Wrap(err, "failed to upload audit log to Microsoft Sentinel")
+	}
+	return nil
+}
+
+func (s sentinel) AuditLoggingEnabled() bool {
+	return true
```

Comment:
Only if someone configures a rule config for audit log collection right?

Reply:
True!

Reply:
Yes, fixed it.